docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -70,37 +70,37 @@ A test that assesses the security of build inputs and outputs with vulnerability
 A component that removes resources that are associated with the completed PipelineRuns. The system assigns resources, such as pods, to every PipelineRun. Without a pruner, these resources remain in the cluster indefinitely, even after the system completes the PipelineRun. 
 
 == Release pipeline 
-A pipeline that defines the process for validating snapshots against the Enterprise Contract. They also provide an array of release destinations other than GitOps deployments. 
+A pipeline that defines the process for validating snapshots against the enterprise contract. They also provide an array of release destinations other than GitOps deployments. 
 
 == Security testing 
 A process that determines if images meet security quality standards.
 
 == Signature and provenance
-Mechanisms that use Tekton chains to extract and store the metadata of the PipelineRuns, sign that metadata, and then store it in the image registry adjacent to the Component image.
+Mechanisms that use Tekton chains to extract and store the metadata of the PipelineRuns, sign that metadata, and then store it in the image registry that is next to the component image.
 
 == Snapshot 
-A set of component and container images that specifies which components should be deployed with which container images. The system creates a snapshot when it finishes running a component's build pipeline. 
+A set of component and container images that specifies which components the system should deploy with which container images. The system creates a snapshot when it finishes running a component's build pipeline. 
 
 == Static environment 
-A set of compute resources that you can use to develop, test, and stage your applications before you release them. You can share static environments across all applications in the workspace. 
+A set of compute resources that you can use to develop, test, and stage your applications before you release them. You can share static environments in all applications in the workspace. 
 
 == {ProductName} 
 A platform to automate the process of building, testing, and deploying applications to the hybrid cloud. {ProductName} offers enterprise-grade security and customizable feature sets.   
 
-== Supply chain Levels for Software Artifacts (SLSA) 
```

Comment:
Please note that although we'd normally do "supply chain," this specific term is named "Supply-chain"